<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WireScanGUI</class>
 <widget class="QWidget" name="WireScanGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>951</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(230, 230, 230);
color: rgb(79, 79, 79);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">QLabel{background: transparent;
             color: rgb(140, 21, 21);
             font-weight: bold;
             font-size: 29px;}</string>
        </property>
        <property name="text">
         <string>LCLS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">QLabel{background: transparent;
color: rgb(79, 79, 79);
Font-size: 11px;}</string>
        </property>
        <property name="text">
         <string>PHYSICS 
APPLICATIONS</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">QLabel{background: transparent;
color: rgb(55, 55, 55);
font-size: 24px;
font-weight: bold;}</string>
        </property>
        <property name="text">
         <string>Wire Scan GUI</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="PyDMLabel" name="PyDMLabel">
           <property name="toolTip">
            <string/>
           </property>
           <property name="channel" stdset="0">
            <string>SIOC:SYS0:AL00:TOD</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PyDMLabel" name="PyDMLabel_2">
           <property name="toolTip">
            <string/>
           </property>
           <property name="channel" stdset="0">
            <string>SIOC:SYS0:AL00:MODE</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QFrame" name="ControlsFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="NavigationgroupBox">
          <property name="title">
           <string>Navigation</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QComboBox" name="beampathCombo"/>
           </item>
           <item>
            <widget class="QComboBox" name="areaCombo"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="MeasurementGroupBox">
          <property name="title">
           <string>Measurement</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QComboBox" name="wireCombo"/>
           </item>
           <item>
            <widget class="QComboBox" name="pmtCombo"/>
           </item>
           <item>
            <widget class="QListWidget" name="bpmList"/>
           </item>
           <item>
            <widget class="QCheckBox" name="JitterCorrection">
             <property name="text">
              <string>Jitter Correction</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="ChargeNormalization">
             <property name="text">
              <string>Charge Normalization</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="ParametersGroupBox">
          <property name="title">
           <string>Parameters</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QLabel" name="InnerLabel">
             <property name="text">
              <string>Inner</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="PyDMCheckbox" name="useXWire">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Use X Wire</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="PyDMLineEdit" name="innerY">
             <property name="toolTip">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="OuterLabel">
             <property name="text">
              <string>Outer</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="PyDMLineEdit" name="innerX">
             <property name="toolTip">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="PyDMLineEdit" name="innerU">
             <property name="toolTip">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="PyDMLineEdit" name="outerX">
             <property name="toolTip">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="ScanPointsLabel">
             <property name="text">
              <string>Scan Points</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="PyDMLineEdit" name="outerY">
             <property name="toolTip">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="PyDMLineEdit" name="outerU">
             <property name="toolTip">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="PyDMLineEdit" name="scanPoints">
             <property name="toolTip">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="PyDMCheckbox" name="useYWire">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Use Y Wire</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="PyDMCheckbox" name="useUWire">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Use U Wire</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="ScanGroupBox">
          <property name="title">
           <string>Scan</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="startButton">
             <property name="text">
              <string>Start Scan</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="abortButton">
             <property name="text">
              <string>Abort Scan</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="plotFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QPushButton" name="saveConfigButton">
        <property name="text">
         <string>Save Config</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="saveDataButton">
        <property name="text">
         <string>Save Data</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="loadDataButton">
        <property name="text">
         <string>Load Data</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="loadConfigButton">
        <property name="text">
         <string>Load Config</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMLabel</class>
   <extends>QLabel</extends>
   <header>pydm.widgets.label</header>
  </customwidget>
  <customwidget>
   <class>PyDMCheckbox</class>
   <extends>QCheckBox</extends>
   <header>pydm.widgets.checkbox</header>
  </customwidget>
  <customwidget>
   <class>PyDMLineEdit</class>
   <extends>QLineEdit</extends>
   <header>pydm.widgets.line_edit</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
